books/Visualization.md

```
@@ -86,6 +86,68 @@ def __init__(self, val=0, next=None):
+```
+
+**Constraints**
+
+```text
+- n == candies.length
+- 2 <= n <= 100
+- 1 <= candies[i] <= 100
+- 1 <= extraCandies <= 50
+```
+
+## Hint
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)